src/theory/theory_model.cpp

```diff
@@ -256,6 +256,18 @@ Node TheoryModel::getModelValue(TNode n, bool hasBoundVars) const
     d_modelCache[n] = ret;
     return ret;
   }
+  // it might be approximate
+  std::map<Node, Node>::const_iterator ita = d_approximations.find(n);
+  if (ita != d_approximations.end())
+  {
+    // If the value of n is approximate based on predicate P(n), we return
+    // choice z. P(z).
+    Node v = nm->mkBoundVar(n.getType());
+    Node bvl = nm->mkNode(BOUND_VAR_LIST, v);
```

Comment:
in the rest of the file, it is kind::BOUND_VAR_LIST.
So, perhaps for readability, we should have the same here.

Reply:
Actually, we are progressively trying to go the opposite direction. :)

Notice that we include namespace `CVC4::kind` at the top of this file.  Thus, the `kind::` is spurious.  There are a number of places in the CVC4 code base where `kind::` is unnecessarily used. As a rule of thumb, all new code I add doesn't use `kind::`. Progressively I'd like to remove all existing unnecessary `kind::` from existing code, but not all at once.